### ISSUE TITLE
Update so that it can use the slim loader

### ISSUE THREAD

**Issue by matthewp** [stealjs/steal-nw#9]
In production mode.

**Comment by matthewp** [comment]
@m-mujica can this be closed?

**Comment by m-mujica** [comment]
@matthewp I don't recall working on this and the fact that there are no comments in the issues makes me think I might have skipped it when I was looking at steal-cordova and friends.

**Comment by matthewp** [comment]
yeah, I think since we switched to electron for our guides we haven't put much work into steal-nw.
